recipes/libsvm/bld.bat

```diff
@@ -0,0 +1,9 @@
+nmake /f Makefile.win all
```

Comment:
Since you have removed the clean command, could you please remove the files in the windows folder before building?